Godeps/Godeps.json

```diff
@@ -3415,6 +3415,10 @@
 		{
 			"ImportPath": "vbom.ml/util/sortorder",
 			"Rev": "db5cfe13f5cc80a4990d98e2e1b0707a4d1a5394"
+		},
+		{
+			"ImportPath": "k8s.io/apimachinery/pkg/util/version",
```

Comment:
Delete this? It's not from the cherry-pick, I added it by hand, and it's not the right commit since I just copied the files over from kubernetes:7ba332a839c.